src/build/hibf/parse_chopper_pack_header.cpp

```diff
@@ -50,7 +52,11 @@ size_t parse_chopper_pack_header(lemon::ListDigraph & ibf_graph,
 
     std::string line;
 
-    std::getline(chopper_pack_file, line); // read first line
+    while (std::getline(chopper_pack_file, line) &&
+           line.size() >= 2 &&
+           line.substr(0, 1) == chopper::prefix::header &&
+           line.substr(1, 1) == chopper::prefix::header_config); // skip config in header
```

Comment:
```suggestion
    while (std::getline(chopper_pack_file, line) &&
           line.size() >= 2 &&
           &line[0] == chopper::prefix::header &&
           &line[1] == chopper::prefix::header_config); // skip config in header
```

`substr` would always construct a new string